src/pages/gd/BuyGD/index.tsx

```diff
@@ -0,0 +1,162 @@
+import React, { memo, useCallback } from 'react'
+import { i18n } from '@lingui/core'
+import { t } from '@lingui/macro'
+import { CentreBox, Converter, GdOnramperWidget, SlideDownTab, Title } from '@gooddollar/good-design'
+import { Box, Text, useBreakpointValue } from 'native-base'
+import { g$Price } from '@gooddollar/web3sdk'
+
+import usePromise from 'hooks/usePromise'
+import useSendAnalyticsData from 'hooks/useSendAnalyticsData'
+
+const BuyGd = memo(() => {
+    const sendData = useSendAnalyticsData()
+
+    const handleEvents = useCallback(
+        (event: string, error?: string) => {
+            sendData({ event: 'buy', action: event, ...(error && { error: error }) })
+        },
+        [sendData]
+    )
+
+    const [G$Price] = usePromise(
+        () =>
+            g$Price()
+                .then(({ DAI }) => +DAI.toSignificant(6))
+                .catch(() => undefined),
+        []
+    )
+
+    const mainView = useBreakpointValue({
+        base: {
+            gap: 40,
+            display: 'flex',
+            flexDirection: 'column',
+            alignItems: 'center',
+            justifyContent: 'center',
+            flexGrow: 1,
+            width: '100%',
+            mb: 2,
+        },
+        lg: {
+            flexDirection: 'row',
+            justifyContent: 'justify-evenly',
+        },
+    })
+
+    const leftContainer = useBreakpointValue({
+        lg: {
+            paddingRight: 40,
+            paddingLeft: 63,
+            alignItems: 'stretch',
+            borderRightWidth: 1,
+        },
+    })
+
+    const rightContainer = useBreakpointValue({
+        lg: {
+            paddingLeft: 40,
+            paddingTop: 60,
+            w: '60%',
+        },
+    })
+
+    const containerCopy = useBreakpointValue({
+        base: {
+            width: 350,
+            fontSize: 'sm',
+        },
+        lg: {
+            width: 600,
+        },
+    })
+
+    const sideTabs = useBreakpointValue({
+        base: {
+            alignItems: 'center',
+        },
+        lg: {
+            alignItems: 'flex-start',
+        },
+    })
+
+    const onrampWrapper = useBreakpointValue({
+        base: {
+            width: '110%',
```

Comment:
whats 110%?

Reply:
needed for a wrapper around the onramp widget. Else on mobile it cuts content within the widget